src/olympia/files/tests/test_file_viewer.py

```diff
@@ -480,8 +484,11 @@ def test_unzip_fatal(self):
     def test_read(self):
         zip_file = SafeZip(self.xpi_path('langpack-localepicker'))
         assert zip_file.is_valid
-        assert 'locale browser de' in zip_file.read('chrome.manifest')
+        assert b'locale browser de' in zip_file.read('chrome.manifest')
 
+    @pytest.mark.skipif(
```

Comment:
Like the other `skipIf` I added this should be double-checked later. https://github.com/mozilla/addons-server/issues/10527